torchmetrics/functional/audio/pit.py

```diff
@@ -153,10 +153,15 @@ def pit(
 
     # calculate the metric matrix
     batch_size, spk_num = target.shape[0:2]
-    metric_mtx = torch.empty((batch_size, spk_num, spk_num), dtype=preds.dtype, device=target.device)
+    metric_mtx = None
     for t in range(spk_num):
         for e in range(spk_num):
```

Comment:
this is not trivial loop anymore, pls use better names than `t` and `e`

Reply:
changed. pls check if it is ok